src/main/java/io/d2a/eeee/converter/StringConverter.java

```diff
@@ -15,6 +15,12 @@
      * @return {exp} in power representation
      */
     public static String toPowString(int exp) {
+        if (exp == 0){
+            return String.valueOf(POWERS[10]);
```

Comment:
```suggestion
            return String.valueOf(POWERS[0]);
```
otherwise returns `⁻` for input 0